lib/index.js

```diff
@@ -73,6 +75,13 @@
  *   Label to use for the footnotes section.
  *   Affects screen reader users.
  *   Change it if you’re authoring in a different language.
+ * @property {string} [footnoteLabelTagName='h2']
+ *   HTML tag to use for the footnote label.
+ *   Can be changed to match your document structure and play well with your choice of css.
+ * @property {Properties} [footnoteLabelProperties={className: ['sr-only']}]
```

Comment:
You need to import TypeScript types, they don’t typically exist as globals.

Somewhere at the top:

```
* @typedef {import('hast').Properties} Properties
```

Reply:
index.js already has this line, but footer.js doesn't. I'll add it there.